sdk/monitor/monitor-opentelemetry-exporter/src/utils/spanUtils.ts

```diff
@@ -38,6 +37,7 @@ function createTagsFromSpan(span: ReadableSpan): Tags {
     const serviceNamespace = span.resource.attributes[SemanticResourceAttributes.SERVICE_NAMESPACE];
     if (serviceName) {
       if (serviceNamespace) {
+        // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
         tags[KnownContextTagKeys.AiCloudRole] = `${serviceNamespace}.${serviceName}`;
```

Comment:
I wonder what're the types of `serviceNamespace` and `serviceName`.

Reply:
Both are `string | number | true`, so I suppose I could just coerce to string